src/main/java/de/timeline/TimelineController.java

```diff
@@ -30,8 +30,10 @@ public void createNewEvent(@RequestBody Event event) {
 		timelineService.createEvent(event);
 	}
 
-	@DeleteMapping("/events")
-	public ResponseEntity deleteOldEvent(@PathVariable UUID uuid) {
+	@DeleteMapping("/events/{uuid}")
+	// hier fehlte die uuid noch als Platzhalter im Mapping, das war sicher im
+	// Video so gezeigt!
```

Comment:
mist ja, hatte da anfangs auch mal ähnlich drin. Aber durch meine vielen Änderungen, habe ich das irgendwann wieder rausgenommen. Aber ich hätte es wahrscheinlich dann in einer falschen Schreibweise gehabt...